ansible/roles/media-licenses/defaults/main.yml

```
@@ -0,0 +1 @@
+SERVER_VIDIS_SYNC_CRONJOB_SCHEDULE: "0/7 * * * *"
```

It shoud run ever 7 minute right?
```suggestion
SERVER_VIDIS_SYNC_CRONJOB_SCHEDULE: "*/7 * * * *"
```